symfony/phpunit-bridge/5.3/bin/phpunit

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env php
+<?php
+if (file_exists(dirname(__DIR__).'/vendor/phpunit/phpunit/phpunit')) {
+    require(dirname(__DIR__).'/vendor/phpunit/phpunit/phpunit');
```

Comment:
This is the new part: if PHPUnit is install directly, use it.

Reply:
I tried adding this to my bin/phpunit file and I'm getting 
```
PHP Fatal error:  strict_types declaration must be the very first statement in the script
```

Reply:
Did you try without strict type declaration?
This file don't include any.

Reply:
Already fixed in https://github.com/symfony/recipes/pull/952

Reply:
> Already fixed in #952

Indeed, thanks